src/getThrow.js

```diff
@@ -0,0 +1,20 @@
+export const ROCK = 'rock';
+export const PAPER = 'paper';
+export const SCISSORS = 'scissors';
+
+
```

Comment:
make sure you delete any unnecessary white space like this! I won't take points off this time since it isn't common in your code, but in the future its something I'm supposed to dock for